src/applications/caregivers/components/FormFields/FacilitySearch.jsx

```diff
@@ -290,6 +298,7 @@ const FacilitySearch = props => {
               value={query}
               onInput={handleChange}
               onSubmit={handleSearch}
+              aria-describedby="facility-search-description"
```

Comment:
May need to confirm that this is valid on the component, since this is a custom component. But it looked like adding the `aria-describedby` would be a good way to link the input and the new text.

Reply:
This won't get passed down into the the shadow DOM to the actual input itself. Would require some SR testing to determine if it does anything with the base level element.

Reply:
That's what I was afraid of 😞

Reply:
I removed the `aria-describedby`